CHANGES.md

```diff
@@ -2,7 +2,8 @@
 - [NEW] `Database` methods `read`, `contains`, `create`, and `delete` now accept local
   (non-replicating documents). These documents must have their document ID prefixed with `_local/`
   and must have their revision ID set to `null` (where applicable).
-  
+- [FIXED] Exception parsing remote database information when replicating with CouchDB 2.3. 
```

Comment:
I think we'll want to say something like `Fixed purge_seq to return a String object when replication with CouchDB 2.3 databases`.
@ricellis Since this may break code for people who except a `long` type, I'm wondering if this should have the `[BREAKING CHANGE]` tag or an additional `Note`.

Reply:
com.cloudant.sync.internal.mazha.CouchDbInfo seems to be an internal domain object that is not exposed through the public API. Shouldn't be used externally. 

Internally, there are not usages of purge_seq attribute.

Reply:
I'm able to import `com.cloudant.sync.internal.mazha.CouchDbInfo`using our [sample todo app](https://github.com/cloudant/sync-android/tree/master/sample/todo-sync).  Someone could be accessing the purge_seq attribute from the remote database using `CouchClient`:
```
        CouchDbInfo ci2 = couchClient.getDbInfo();
        long purgeSeq = ci2.getPurgeSeq();
```

Reply:
@emlaver the [documentation](https://github.com/cloudant/sync-android/blob/master/doc/migration.md) states:
>All private API classes are in the com.cloudant.sync.internal package. API users should not use these classes as fields, method signatures, and implementation details may be subject to change. Directly using the classes in these packages or calling their methods is not supported.

so I think we are safe regarding a breaking change - that said we do need to support deserialization from earlier versions that do return the `long` - it looks like this change does achieve that though.